components/bunnygrabber.py

```diff
@@ -1,19 +1,31 @@
+# import Robot py library so this file can interface with the robot.
 import wpilib
 
-
+# Class for controlling arm, will be instantiated in later script usage.
```

Comment:
This doesn't control the whole arm, just the grabber at the end. This should become part of the classes docstring directly after the start of the class, also the comment about instantiating the class is unneeded.